roles/slurm_management/templates/cron_add_missing_slurm_accounts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Search for all existing groups inside the '/groups' folder
+# that are inside /mnt/*/ folder on the SAI machines
+for _each_pfs in /mnt/*; do
+   cd ${_each_pfs}/groups
+   # Find should be pretty robust way, but you need to manually
+   # remove the leading dot slash from from the group name: "./umcg-atd"
+   _accounts_list="$(find . -maxdepth 1 -mindepth 1 -type d | sed "s/^\.\///g")"
+   
+   IFS=$'\n'
+   for _each_group in "${_accounts_list}"; do
```

Comment:
```suggestion
   readarray -t _groups < <(find . -maxdepth 1 -mindepth 1 -type d | sed "s/^\.\///g")
   for _each_group in "${_groups[@]}"; do
```

Reply:
I was unaware of the readarray. Thanks, I already like it.